Flow.Launcher/Converters/BorderClipConverter.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Data;
+using System.Windows.Media;
+using System.Windows.Documents;
+using System.Windows.Shapes;
+
+// For Clipping inside listbox item
+
+namespace Flow.Launcher.Converters
+{
+    public class BorderClipConverter : IMultiValueConverter
+    {
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (values.Length == 3 && values[0] is double && values[1] is double && values[2] is CornerRadius)
```

Comment:
i have been playing just mainly switching the themes around in settings and sometimes i get a null exception on a random theme which i think maybe the `values` could be null. Will have to have another play to know how to repro.